doc/doc-requirements.txt

```diff
@@ -20,5 +20,5 @@ Jinja2
 sphinx-design>=0.5.0
 boto3
 tensorflow
-pytorch
```

Comment:
yes, I should have done it in a new PR. 

I just spot a typo. in pypi, the name in torch where in conda package it is pytorch. Since this requirement file is for pypi, I updated it to help the doc generation

Reply:
should I remove it and create a new PR ?